sdk/resources/mgmt/src/main/java/com/azure/management/resources/Deployment.java

```diff
@@ -263,6 +264,8 @@
             Deployment beginCreate();
 
             Mono<Deployment> beginCreateAsync();
+
+            AcceptedCreatable<DeploymentExtendedInner, Deployment> acceptCreate();
```

Comment:
It is for `PollerFlux<PollResult<InnerT>, InnerT>`. If we do not provide that, we can avoid this as return type.

Reply:
We might first not provide the PollerFlux. Let us see whether it is useful enough (it could provide a Flux of PollResult, which provides status of `InProgress` or `Success`, but the actual type could be of little use since we do not have much idea on the response type during polling).